### ISSUE TITLE
no-duplicate-at-import-rules not working for less file

### ISSUE THREAD

**Issue by Evangelink** [stylelint/stylelint#3416]
Hi there,

I have noticed that the rule `no-duplicate-at-import-rules` doesn't raise on a less file for such kind of cases:

```
@import "a.css";
@import "a.css";
```

or even with the code in the unit-tests:

```
@import "a.css" screen; @import "b.css" tv; @import "a.css" tv;
```

I have noticed this is working properly when the file extension is either `.css` or `.scss` , only failing for `.less`. I checked the implementation and the unit tests and it seems to be handled and covered correctly.

Is there anything special about this rule and less files?

Cheers,
Amaury

**Comment by jeddy3** [comment]
> Is there anything special about this rule and less files?

Yes, this rule is not compatible with `@import` in Less file. This is documented in the [rule's README](https://stylelint.io/user-guide/rules/no-duplicate-at-import-rules/).

The built-in rules are geared towards standard syntax. However, they will try to be compatible with non-standard syntax _when the specific parser allows for it_. In this instance, the Less parser treats `@import` as a special node type, so the rule ignores them.

In these instances, we recommend [writing a plugin](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/plugins.md) to geared towards the particlar syntax. This is what [stylelint-scss](https://github.com/kristerkari/stylelint-scss) does in some of its rules.
